test/core/models/ocelot/LabelSpec.scala

```diff
@@ -455,4 +468,64 @@ class LabelSpec extends BaseSpec with ProcessJson {
       labels3.value("Location") shouldBe Some("Here")
     }
   }
+
+  "Labels Flow stack" must {
+    "Allow adding a Flow to top of stack" in {
+      val labels = LabelCache().pushFlows(Seq("1","2"), "3", Some("loop"), Seq("One", "Two", "Three"), Nil)
+
+      labels.stackList.length shouldBe 3
+      labels.stackList.head shouldBe Flow("1", Some(LabelValue("loop", Some("One"))))
+      labels.stackList(1) shouldBe Flow("2", Some(LabelValue("loop", Some("Two"))))
+      labels.stackList(2) shouldBe Continuation("3", Nil)
+    }
+
+    "Allow removal of Flow from top of stack" in {
+      val labels = LabelCache().pushFlows(Seq("1","2"), "3", Some("loop"), Seq("One", "Two", "Three"), Nil)
+
+      labels.takeFlow.map{t =>
```

Comment:
stanzas is not used so could be replaced with "_"

Reply:
done